.github/workflows/validation.yml

```diff
@@ -99,6 +99,31 @@ jobs:
           cd framework/pipeline/fileopening
           restManager PrintTrees v2.2.30_hits.root
 
+  macros-health:
+    name: "Macros with clean error output"
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/lobis/root-geant4-garfield:rest-for-physics
+    needs: [ framework-install ]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: rest-for-physics/framework
+          path: framework
+      - name: Restore cache
+        uses: actions/cache@v3
+        id: framework-install-cache
+        with:
+          key: ${{ env.BRANCH_NAME }}-${{ github.sha }}
+          path: ${{ env.REST_PATH }}
+      - name: Load restRootMacros
+        run: |
+          source ${{ env.REST_PATH }}/thisREST.sh
+          cd framework/pipeline
+          export DISPLAY=localhost:0.0
```

Comment:
Why this line is needed?

Reply:
When there is no DISPLAY defined then I get a warning since I do not have proper X-forwarding configured. This generates a warning on my side, locally. I thought it could be a cause of troubles also at the docker. But I imagine it is not.

```
jgalan@sultan2:~/rest-framework/source/libraries$ restRootMacros 
Warning in <UnknownClass::SetDisplay>: DISPLAY not set, setting it to 192.168.1.2:0.0
```

Reply:
I don't think this is an issue in the docker but perhaps @lobis can comment on it.

This line seems weird to me, I rather prefer to remove it.

Reply:
There is no issue on the docker, and I don't think that the line seems weird to you is a solid argument to take out the line.

Setting that environment variable there is completely harmless, and it only has effect on that particular validation test. 

Don't see a real problem here, but. if you give me a good reason to take out the line, I will take it out.

Reply:
I think adding the DISPLAY env variable is the easiest way to avoid the problem but I guess the most correct thing to do would be to load `restRoot` using batch mode or something similar, so that it doesn't care about displays.

I would think that the `-b` flag would solve this but apparently not so its probably not easy or possible to fix this warning, so I would keep it unless someone has a better idea.

Reply:
@lobis do you think it is possible to solve the Garfield output warnings? This might be related to a Garfield not compiled with the same ROOT version? I do not have those messages locally.

Reply:
> @lobis do you think it is possible to solve the Garfield output warnings? This might be related to a Garfield not compiled with the same ROOT version? I do not have those messages locally.

There is only one root version installed on the image, and Garfield is installed after root (and with the `PATH`, `ROOTSYS` and `LD_LIBRARY_PATH` variables correctly referencing the root install. @juanangp and I tried to remove these warnings extensively for a while but didn't manage to do it.

If you want to try I think the easiest way would be to install Garfield and then REST on the official root docker image (`docker run --rm -it rootproject/root`) and see if you can fix it.

Reply:
Probably the problem is that we are not adding the Garfield headers to `ROOT_INCLUDE_PATH`